charts/gofer/templates/deployment.yaml

```diff
@@ -39,7 +39,17 @@ spec:
           {{- if .Values.configHcl }}
           args: [ "-c", "/config/gofer.hcl", "{{ .Values.goferMode }}", "-v", "{{ .Values.logLevel | default "debug" }}"]
           {{- else }}
-          args: ["{{ .Values.goferMode }}", "-v", "{{ .Values.logLevel | default "debug" }}"]
+          args:
```

Comment:
Maybe we could keep consistency between https://github.com/chronicleprotocol/charts/blob/main/charts/spectre/templates/deployment.yaml#L43

and I would not mind having [this](https://github.com/chronicleprotocol/charts/blob/main/charts/spectre/templates/deployment.yaml#L36) also added

Reply:
I've used the spectre patter on bot now, check it out